soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -283,6 +283,18 @@ bool IsValidSaveFile() {
     return validSave;
 }
 
+bool HasSong(ItemTrackerItem item) {
+    return (1 << item.id) & gSaveContext.inventory.questItems;
+}
+
+bool HasQuestItem(ItemTrackerItem item) {
+    return (item.data & gSaveContext.inventory.questItems) != 0;
+}
+
+bool HasEqItem(ItemTrackerItem item) {
+    return (item.data & gSaveContext.inventory.equipment) != 0;
+}
```

Comment:
EqItem feels a little vague to me. What if this would be called HasEquipment instead?

Reply:
I mean, Eq is almost always Equipment in a game, but I'm not opposed.

Reply:
I agree, either `HasEquipItem` or `HasEquipment` feel better and have the same length as `HasQuestItem`.

Reply:
Done. I chose `HasEquipment` as that follows the `HasSong` precedent of not requiring "item" in the name.